source/extensions/filters/network/redis_proxy/proxy_filter.cc

```diff
@@ -57,6 +57,7 @@ void ProxyFilter::initializeReadFilterCallbacks(Network::ReadFilterCallbacks& ca
                                                config_->stats_.downstream_cx_tx_bytes_total_,
                                                config_->stats_.downstream_cx_tx_bytes_buffered_,
                                                nullptr, nullptr});
+  splitter_.setDispatcher(callbacks.connection().dispatcher());
```

Comment:
re: my other comment about passing the dispatcher into the constructor, maybe use a splitter factory in the config which can be called here to construct the splitter?

Reply:
I'll create a splitter factory in `config.h/cc`; essentially it will move a lot of the config code into a `create` method due to the fact the splitter depends on several unique pointers as inputs.